LOCAL.md

```diff
@@ -0,0 +1,48 @@
+# Local development
+
+The fastest and easiest way to start up is to use docker-compose.develop which gives developers the ability to start the bot up with all the required dependencies, *without* needing to install any freqtrade specific dependencies on your local machine.
+
+# Install
+## git
+``` bash
+sudo apt install git
+```
+
+## docker
+``` bash
+sudo apt install docker
+```
+
+## docker-compose
+``` bash
+sudo apt install docker-compose
```

Comment:
Can we maybe replace these `apt` calls with a link to the docker-documentation and a "make sure to have docker and docker-compose installed"? 

This will work on ubuntu/debian (and install old versions - at least on debian), but will fail on Windows, MacOS, and any other flavor of linux. Having comprehensive documentation on how to install docker should not be part of the freqtrade documentation i think...